### PR TITLE
Polish translation update

### PR DESCRIPTION
**Pull request by archeYR**

### CODE REVIEW

**Comment by HBelusca** [comment]
Two things to notice:
- your branch has conflicts, due to CR/LF problems: you should fix that first;
- then all of your commits should be squashed before being included in ROS.

**Comment by learn-more** [comment]
And another thing to note: All your commits are under the name 'archeYR', and we require a real name instead of a nickname.

File: base/applications/charmap_new/lang/pl-PL.rc [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-    IDS_TITLE "Character Map"
-END
+LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
+
+IDD_CHARMAP DIALOGEX 6, 6, 290, 224
+FONT 8, "MS Shell Dlg", 0, 0
+STYLE WS_OVERLAPPED | WS_CAPTION | WS_SYSMENU | WS_MINIMIZEBOX | WS_SIZEBOX
+CAPTION "Tablica znaków ReactOS"
+BEGIN
+    LTEXT "Czciąka:", IDC_STATIC, 6, 7, 24, 9
```
Comment (Saibamen) [inline base/applications/charmap_new/lang/pl-PL.rc]: Czcionka

File: base/applications/msconfig_new/lang/pl-PL.rc [hunk 2]
```diff
@@ -1,247 +1,247 @@
...
+    CONTROL         "List1", IDC_TOOLS_LIST, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 5, 5, 356, 118
+    EDITTEXT        IDC_TOOLS_CMDLINE, 5, 139, 356, 14, ES_READONLY
+    PUSHBUTTON      "&Uruchom", IDC_BTN_RUN, 311, 156, 50, 14
+    CONTROL         "&Opcje zaawansowane", IDC_CBX_TOOLS_ADVOPT, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 5, 158, 134, 12
+    LTEXT           "Wybrane &polecenie:", IDC_STATIC, 5, 127, 128, 10
+END
+
+IDD_FILE_EXTRACT_DIALOG DIALOGEX 0, 0, 353, 117
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
+CAPTION "Rozwiń pliki z źródła instalacji"
```
Comment (Saibamen) [inline base/applications/msconfig_new/lang/pl-PL.rc]: Rozwiń pliki ze źródła instalacji

**Comment by archeYR** [comment]
I'm trying to. Sorry, I'm newbie in git and I still have some issues with it.

**Comment by tkreuzer** [comment]
@archeYR That's normal when you are new to Git. You can either go the easy way (create a new branch, get your changes into it somehow, perhaps just copy your file out and copy it back in the new branch, then create a new PR) or - and I suggest that for learning reasons - rebase your branch: First switch to your branch in your working copy. If you use TortoiseGit, right click on the root folder, select TortoiseGit->rebase..., then in the upper right select master as the branch to rebase upon. Now you should see the list of your commits. Mark all, except the one at the bottom (this is your 1st commit in the branch), right click on a marked commit and select "squash with commit below", which will squash everything you did into a single commit. Finally click on the rebase button. Now it should do some stuff and present you a window that contains all your commit messages. Just delete everything and type in the right commit message, then confirm, and if all worked, you can press the "Done" button and then push the branch again (you will need to check the "Force: ... [ ] known changes" checkbox in the commit dialog, since you overwrite your old branch, but that is ok.) If something went wrong and you get a merge conflict or some other error, just press the abort button and go back to the easy method. If you don't use TortoiseGit, .... RTFM :P

**Comment by archeYR** [comment]
Thank you, unfortunately it didn't worked, I will create a new pull request soon, let's forget about anything what was happening here. Sorry.

**Pull request closed without merging**